### PR TITLE
[shiftonstack] Type 3 - external ceph support

### PR DESCRIPTION
**Pull request by psathyan**
Deployed Topology 03 type for OpenShift on OpenStack.

Depends-On: https://github.com/openstack-k8s-operators/ci-framework/pull/1729

### CODE REVIEW

File: automation/vars/shiftonstack.yaml [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+# Stores the different automation workflow steps required by various
+# shiftonstack deployed topologies.
+vas:
+  shiftonstack03:
```
Comment (rlobillo) [inline automation/vars/shiftonstack.yaml:5]: would you mind to rename to `shiftstack` ?

**Comment by psathyan** [comment]
recheck

**Comment by psathyan** [comment]
recheck

File: automation/vars/shiftonstack.yaml (hunk 1 above)
Comment (psathyan) [reply automation/vars/shiftonstack.yaml]: Done

**Comment by psathyan** [comment]
recheck

**Comment by fultonj** [comment]
@queria would you please take over this PR?

**Comment by fultonj** [comment]
No response. Closing. It can be reopened if really necessary.

**Pull request closed without merging**